### ISSUE TITLE
Bug: can't import and run sample

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [mike14u/croperino#37]
I get this:

![image](https://user-images.githubusercontent.com/5357526/47465873-68350800-d7f7-11e8-9501-ff350644f493.png)

**Comment by mike14u** [comment]
Hmm, should work though since its a local copy for you.

**Comment by AndroidDeveloperLB** [comment]
I don't understand. Can you please check it out?

**Comment by mike14u** [comment]
@AndroidDeveloperLB https://stackoverflow.com/questions/52424269/illegalaccessexception-can-not-access-a-member-of-class-com-android-build-gradl

Can you try to invalidate caches? It doesn't happen on mine in multiple machines.

**Comment by AndroidDeveloperLB** [comment]
I've tried now on a different PC (laptop), and it worked fine. Odd.
But, I've tried to open a large image, and it got a very bad quality.

Attached here a large image:

[original.zip](https://github.com/mike14u/croperino/files/2513911/original.zip)

I've also noticed that after I've chosen to crop, it removed the original image. 
Originally, I wanted to send you the image of this:
https://www.wallpaperup.com/489485/Awesome_Bridge.html

But as I wrote, it was replaced by a cropped version of it. Not nice.
